app/knowledge_panels.py

```diff
@@ -1,8 +1,12 @@
+import imp
```

Comment:
Deprecated since version 3.4: The [imp](https://docs.python.org/3/library/imp.html#module-imp) module is deprecated in favor of [importlib](https://docs.python.org/3/library/importlib.html#module-importlib).

https://docs.python.org/3/library/imp.html

Reply:
Its unused import, so removed :)